ui/package.json

```diff
@@ -38,7 +38,8 @@
     "vitest": "2.1.2"
   },
   "dependencies": {
-    "@algorandfoundation/algokit-utils": "8.0.3",
+    "@algorandfoundation/algokit-utils": "9.1.2",
+    "@algorandfoundation/xgov": "github:algorandfoundation/xgov-beta-ts#chore/upgrade-to-algosdk-3",
```

Comment:
Could you publish a pre-release for this instead of sourcing the code from a GitHub branch? Even a `.alpha` release or something would be preferable, effectively "locking in" the source code (and preventing someone from accidentally pushing a commit that breaks the app).

I'm extremely hesitant to deploy this to production if it can be avoided.

Reply:
absolutely! apologies, this was supposed to be temporary